lib/network/modules/PhysicsEngine.js

```diff
@@ -121,14 +121,13 @@ class PhysicsEngine {
       this.stopSimulation(false);
       this.body.emitter.off();
     });
-    // this event will trigger a rebuilding of the cache everything. Used when nodes or edges have been added or removed.
     this.body.emitter.on("_dataChanged", () => {
-      // update shortcut lists
+      // Nodes an/or edges have been added or removed, update shortcut lists.
```

Comment:
`an/or` => and/or

Reply:
OK. Can you hold off merging until @mbroad has reviewed, please?

Reply:
Of course, that's exactly why I waited